static/scripts/fetch-templates.js

```diff
@@ -0,0 +1,146 @@
+var rp = require('request-promise');
+const yaml = require('js-yaml');
+const fs   = require('fs');
+
+var templateRepoRoot = "https://raw.githubusercontent.com/platformsh/template-builder/master/templates/";
+var infoFile = ".platform.template.yaml";
+var appYAML = "files/.platform.app.yaml";
+
+const finalFileLocation = "data/templates.yaml";
+// const finalFileLocation = "test.yaml";
+
+var options = {
```

Comment:
```suggestion
let options = {
```
Or possibly `const`.  But `var` isn't recommended these days in JS, AFAIK.  (`let`'s scoping rules are different and less prone to getting confused.)